config/default.yml

```diff
@@ -4669,6 +4669,11 @@ Style/MethodDefParentheses:
     - require_no_parentheses
     - require_no_parentheses_except_multiline
 
+Style/MethodDefined:
+  Description: 'Checks for usages of `instance_methods` that can be replaced with `method_defined?`.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
```

Comment:
I have considered marking the cop as unsafe, but didn't for now because `instance_methods` seems like such an essential method, and the `instance_methods.include?` pattern is unlikely to produce false positives. What do you think?